### ISSUE TITLE
[iOS] Cant create archive on XCode 11

### ISSUE THREAD

**Issue by yestay90** [PSPDFKit/react-native#326]
# Details
My RN version is 60.5. I am adding this framework manually following to instructions. 
App is working on simulator but when trying to create archive 
`
node_modules/react-native-pspdfkit/ios/RCTPSPDFKit/RCTPSPDFKitView.h:13:9: fatal error: module 'PSPDFKit' not
      found
@import PSPDFKit;
 ~~~~~~~^~~~~~~~
1 error generated.
`
I looks that file in RCTPSPDFKit.xcodeproj doesnt see PSPDFKit at all. 

# Acceptance Criteria

**Comment by radazzouz** [comment]
Hi @yestay90,

I am sorry to see that you're experiencing integration issue.

It's very hard for us to know for sure what's going on since we have no visibility into your project's setup or your development environment.

PSPDFKit 9, unlike previous versions, uses XCFrameworks instead of fat `.frameworks`, so the integration process has changed from the manual integration to CocoaPods. Can you please try and use the new [integration process](https://github.com/PSPDFKit/react-native#getting-started.) instead?

Also if, for some reason, you absolutely need to use the old manual integration, please make sure that the XCFrameworks are properly added to your app. 

<img width="1343" alt="Screen Shot 2019-11-11 at 7 48 23 AM" src="https://user-images.githubusercontent.com/7443038/68588514-b7c49100-0457-11ea-8369-4fc208209b8f.png">

See https://pspdfkit.com/guides/ios/current/getting-started/integrating-pspdfkit/#integrating-the-xcframework for more details.

I just tested archiving our [Catalog sample project](https://github.com/PSPDFKit/react-native#running-catalog-project) and it worked without any issues.

<img width="754" alt="Screen Shot 2019-11-11 at 7 51 59 AM" src="https://user-images.githubusercontent.com/7443038/68588764-4b965d00-0458-11ea-9667-65be242e1077.png">

Hope that this helps!

**Comment by yestay90** [comment]
Hi @radazzouz 
We switched to CocoaPod installation. Can you help us we have error:
```error: Multiple commands produce /BuildProductsPath/Release-iphoneos/PSPDFKitUI.framework.dSYM':
error: Multiple commands produce /Frameworks/PSPDFKitUI.framework
error: Multiple commands produce Frameworks/PSPDFKit.framework'
error: Multiple commands BuildProductsPath/Release-iphoneos/PSPDFKit.framework.dSYM':```

**Comment by radazzouz** [comment]
@yestay90, I am still not 100% sure what's going on in your project, as I am unable to reproduce this on my side in the Catalog app.

I suspect that this is caused by not completely removing the old integration and having our plugin added both like before and via CocoaPods.

I googled the error message and I found these Stack Overflow topics: 

- https://stackoverflow.com/questions/52452472/xcode-10-error-multiple-commands-produce-react-native
- https://stackoverflow.com/questions/50718018/xcode-10-error-multiple-commands-produce/50719379#50719379

Let me know if they help.

**Comment by yestay90** [comment]
@radazzouz 
When installing PSPDFKit using CocoaPods should we add libRCTPSPDFKit.a to embedded library list?

**Comment by radazzouz** [comment]
No, this is no longer required since [React Native 0.60.x when CocoaPods is used by default](https://facebook.github.io/react-native/blog/2019/07/03/version-60#cocoapods-by-default).

Please make sure that you've added our plugin correctly into your app by using CocoaPods, as indicated in step 6 our our [integration guide](https://github.com/PSPDFKit/react-native#getting-started).

**Comment by wbercx** [comment]
For what it's worth, I ran into the same and realised I was using an older version of Cocoapods. Updating that and then reinstalling my pods resolved the issue.

**Comment by radazzouz** [comment]
Thanks for sharing @wbercx!

@yestay90 are you still experiencing integration issues?
